azure.yaml

```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/alpha/azure.yaml.json
+
+name: asa-samples-web-application
+metadata:
+  template: asa-samples-web-application@0.0.1-beta
+services:
+  simple-todo-web:
+    project: ./src/web
+    host: springapp
+    language: java
+    spring:
+      deploymentName: default
+    hooks:
+      prepackage:
+        windows:
+          shell: pwsh
+          run: cd ../client;mvn clean install
```

Comment:
@fangjian0423  it just came to me, should I replace the `mvn` commands here and below with maven wrapper in the root directory, to avoid adding maven as the prerequisite?

Reply:
> @fangjian0423 it just came to me, should I replace the `mvn` commands here and below with maven wrapper in the root directory, to avoid adding maven as the prerequisite?

yep, replace it.